Automattic/wp-calypso
# Add navigation for migration instruction steps

Related to #91777 and #91852
Based off of #92070

## Proposed Changes

* Add the navigation logic for the migration instructions steps.
  * Notice that it doesn't register events yet. It will be done as part of [this issue](https://github.com/Automattic/wp-calypso/issues/91774).

## Testing Instructions



* Navigate to `/start`.
* Navigate through the flow to migrate a site.
* When you go to the migration instructions step, make sure you can navigate between the steps properly.
* And when you complete it, make sure you are redirected to the migration started step.

## Recording

https://github.com/Automattic/wp-calypso/assets/876340/cf28d121-5a9c-45ee-b085-458e51db98ff

## Screenshots

<img width="467" alt="Screenshot 2024-07-15 at 10 15 49" src="https://github.com/user-attachments/assets/7e4dd1c3-9254-4749-8624-9b0ba4d4ae15">

<img width="466" alt="Screenshot 2024-07-15 at 10 16 00" src="https://github.com/user-attachments/assets/5ac2d318-7999-4307-b966-acd1d61ad22d">

<img width="462" alt="Screenshot 2024-07-15 at 10 16 07" src="https://github.com/user-attachments/assets/70a45282-4b22-46f7-a505-0e3bdd31da12">

## Pre-merge Checklist
